DEVDOCS-2086: Dev Center | Theme Docs | Theme Variations (#624)
bigcommerce/dev-docs: docs/stencil-docs/deploying-a-theme/naming-your-theme.md

<div class="otp" id="no-index">

### On this page
- [Naming your theme](#naming-your-theme)
- [Creating and naming a new theme variation](#creating-and-naming-a-new-theme-variation)
- [Changing a theme variation’s font family](#changing-a-theme-variations-font-family-configjson)
- [Preparing a variation’s thumbnails](#preparing-a-variations-thumbnails)
- [Resources](#resources)

</div> 

## Naming your theme

> The basic details of the Cornerstone theme are outlined in the theme's configuration file, [`config.json`](https://github.com/bigcommerce/cornerstone/blob/master/config.json). You can update the Cornerstone's `config.json` file to reflect your theme's values such as name, version number, and documentation URL. 

Reviewer: ```suggestion
The basic details of the Cornerstone theme are outlined in the theme's configuration file, [`config.json`](https://github.com/bigcommerce/cornerstone/blob/master/config.json). You can update the Cornerstone's `config.json` file to reflect your theme's values, such as name, version number, and documentation URL. 

```

```json
{
  "name": "Cornerstone",
  "version": "4.9.0",
  "meta": {
    "price": 0,
    "documentation_url": "https://support.bigcommerce.com/articles/Public/Cornerstone-Theme-Manual",
  // ...
  }
}
```

> The following example shows how to change Cornerstone's configurations to reflect your own theme’s name, version number, price on Theme Marketplace, and documentation URL:

Reviewer: ```suggestion
The following example shows how to change Cornerstone's configurations to reflect your own theme's name, version number, price on Theme Marketplace, and documentation URL:

```

Reviewer: Use straight apostrophes, not the curly ones.

}
```

## Creating and naming a new theme variation

> You define variations in your theme's `config.json` file. Below is the Cornerstone Light variation’s definition from Cornerstone's `config.json`:

Reviewer: ```suggestion
You define variations in your theme's `config.json` file. Below is the Cornerstone Light variation's definition from Cornerstone's `config.json`:

```

-->

```json
  "variations": [
    {
      "name": "Light",
      "id": "light",
      "meta": {
        "desktop_screenshot": "desktop_light.png",
        "mobile_screenshot": "mobile_light.png",
        "description": "Ideal for a wide range of businesses and brands, this design is fully responsive, simple, and ready for you to add your branding, logo, and products. It comes ready to build a clean looking and beautiful store with features such as homepage carousel, social media icons, featured and top selling products, and faceted search (which is available on select BigCommerce plans). Fully express your brand by using our Theme Editor to quickly and easily style your site's fonts and colors, all without the need to write code.",
        "demo_url": "https://cornerstone-light-demo.mybigcommerce.com",
        "optimized_for": [
          "multi_purpose",
          "mobile_tablet_desktop",
          "sales_discounts",
          "large_catalog"
        ],
        //...
      },
      //..
    },
```

> Try inserting your custom variation into the `variations` object of Cornerstone's `config.json`. Your variation’s definition might look something like this:

Reviewer: ```suggestion
Try inserting your custom variation into the `variations` object of Cornerstone's `config.json`. Your variation's definition might look something like this:

```

      "industries": []
    },
  },
  // ...
```

> ## Changing a theme variation’s font family

Reviewer: ```suggestion
## Changing a theme variation's font family

```

}
```

> ## Preparing a variation’s thumbnails

Reviewer: ```suggestion
## Preparing a variation's thumbnails

```

For details on preparing screenshots that enable merchants to preview each variation, and to preview your theme as a whole in the BigCommerce control panel and Theme Marketplace, see [Preparing Thumbnail Images](https://developer.bigcommerce.com/stencil-docs/deploying-a-theme/preparing-thumbnail-images).

<div class="HubBlock--callout">
<div class="CalloutBlock--">
<div class="HubBlock-content">

<!-- theme:  -->

### Using Stencil CLI to change variations

> > To locally test/debug a specific variation of your theme, launch it by invoking the Stencil CLI’s `stencil start -v` switch. Enter the following code into the terminal, replacing `variation-name` with your theme's variation name as listed in `config.json`:

Reviewer: ```suggestion
> To locally test/debug a specific variation of your theme, launch it by invoking the Stencil CLI's `stencil start -v` switch. Enter the following code into the terminal, replacing `variation-name` with your theme's variation name as listed in `config.json`:

```
